governance/votes/Executive vote - August 18, 2023.md

```diff
@@ -0,0 +1,225 @@
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Dai Savings Rate will be reduced per the latest update to the EDSR, as detailed below.
+- Multiple stability fee updates will be made, as detailed below.
+- The Smart Burn Engine parameters will be updated, as detailed below.
+- Vault parameter updates per the Stability Scope will be made, as detailed below.
+- CRVV1ETHSTETH-A offboarding will be completed, as detailed below.
+- A total of **272.75 MKR** will be paid to sixteen Aligned Delegates, as detailed below.
+- Housekeeping parameter updates to the old Compound and Aave Direct Deposit Modules will take place, as detailed below.
+- Parameter updates to the New Silver (RWA002-A) vault will be made, as detailed below.
+- Admin control over the remaining Spark Protocol contracts will be transferred to the Spark Proxy, as detailed below.
+- A proxy spell for Spark Protocol will be triggered, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Update
+
+As per this successful governance [poll](TBD), the DSR will be reduced as per the latest updates to the [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-) in the Stability Scope as follows, if this executive proposal passes.
+
+- Reduce the DSR by 3% from 8% to **5%**.
+
+Please review the forum [thread](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-modify-enhanced-dsr-based-on-observed-data/21581) to inform your position before voting.
+
+### EDSR-Related [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Updates
+
+As per this successful governance [poll](TBD), the following Stability Fee Updates will be made, as per the instructions for the [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-) in the Stability Scope, if this executive proposal passes.
```

Comment:
Slightly confusing because in MIP104, it says "Other parameters and mechanisms that are dependent on the DSR, such as certain stability fees, are not affected by the EDSR, but only by the underlying DSR"